main.go

```diff
@@ -119,7 +135,7 @@ func realMain(echoChan chan<- *echo.Echo) int {
 	return exitCode
 }
 
-func setupDeliverer(httpClient *http.Client, logger *log.Logger, buildVersion string) (*queue.Deliverer, error) {
+func setupDeliverer(httpClient *http.Client, logger *log.Logger, manager *shared.PluginManager, buildVersion string) (*queue.Deliverer, error) {
 	sharedKey := os.Getenv("HSDP_LOGINGESTOR_KEY")
```

Comment:
Are these variables used anywhere? I was looking for places where there is a null/empty check is done (in case it is not set the app should not start IMO)

Reply:
This is a helper function. It is only called when all parameters have a known good state i.e. in case of null logproxy would have bailed already -- or it `nil` could be a valid value as well e.g. missing. The linter does a good job of warning about "might be nil"